sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -1,10 +1,9 @@
 # Release History
 
-## 7.0.6 (Unreleased)
+## 7.0.6 (2021-05-11)
 
 ### New Features
 
-- Enable encoding the body of a message to the 'value' or 'sequence' sections (via AmqpAnnotatedMessage.bodyType). Using this encoding is not required but does allow you to take advantage of native AMQP serialization for supported primitives or sequences.
 - Adds support for passing `NamedKeyCredential` as the credential type to `ServiceBusClient` and `ServiceBusAdminstrationClient`. Also adds support for passing `SASCredential` to `ServiceBusClient`.
```

Comment:
This is technically "new feature". So, either update the package version to 7.1.0 or hide this feature as well

Reply:
Updated to 7.1.0.

CC: @KarishmaGhiya